src/notation/view/widgets/editstyle.ui

```diff
@@ -12053,7 +12078,7 @@ first note of the system</string>
                       <string>Reset to default</string>
                      </property>
                      <property name="accessibleName">
-                      <string>Reset 'Melisma pad' value</string>
+                      <string>Reset 'Line gap' value</string>
```

Comment:
to match the actual string in the settings ("Melisma" and "gap" were wrong, and "Underscore" just got changed to "Extension line", here though "Line" is suficcient)